modules/aws/errors.go

```diff
@@ -60,3 +60,20 @@ func (err AsgCapacityNotMetError) Error() string {
 func NewAsgCapacityNotMetError(asgName string, desiredCapacity int64, currentCapacity int64) AsgCapacityNotMetError {
 	return AsgCapacityNotMetError{asgName, desiredCapacity, currentCapacity}
 }
+
+// BucketVersioningNotEnabledError is returned when an S3 bucket that should have versioning does not have it applied
+type BucketVersioningNotEnabledError struct {
+	s3BucketName     string
+	awsRegion        string
+	versioningStatus string
+}
+
+func (err BucketVersioningNotEnabledError) Error() string {
+	return fmt.Sprintf(
+		"Versioning status for bucket #{err.s3BucketName} in the #{err.awsRegion} region is #{err.versioningStatus}",
```

Comment:
Ah I just realized this after merging. I don't think go actually supports string interpolation like this. Can you update this in your next PR?

Reply:
I noticed some functions in that file already that used that syntax and made an (errant) assumption that this was some supported syntax I hadn't stumbled upon.  Sad day for me :( 

Looking at my latest merge from master, I see that syntax has all been corrected.  Holdover from Ruby I guess.  I will fix up.